buildenv/clang/Dockerfile

```diff
@@ -3,10 +3,8 @@ FROM autopas/build-base
 ENV CLANGVERSION 9
 
 # add llvm repository
-RUN echo "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-6.0 main\n\
-deb-src http://apt.llvm.org/bionic/ llvm-toolchain-bionic-6.0 main\n" > /etc/apt/sources.list.d/llvm.list
```

Comment:
I removed thos because they looked unused to me. Might be a legacy thing because we used clang-format-6 at the beginning (not even sure about that)?